dev-python/elasticsearch-py/elasticsearch-py-6.0.0.ebuild

```diff
@@ -40,6 +40,17 @@ DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
 		dev-python/nosexcover[${PYTHON_USEDEP}]
 		virtual/jre:1.8 )"
 
+python_prepare_all() {
```

Comment:
I should probably complain that you're mixing a version bump with bugfix to older versions (whose purpose I don't really understand given that the dep is still the same).